find_green_build

```diff
@@ -122,7 +125,8 @@ fi
 
 logecho
 release::set_build_version $TESTING_BRANCH $LOCAL_CACHE \
-                           "$FLAGS_exclude_suites" || common::exit 1
+                           "$FLAGS_exclude_suites" \
+                           $FLAGS_limit || common::exit 1
```

Comment:
Yes.  The function itself still needs a reasonable limit in case it's called and that arg isn't set.  PTAL.